eng/native/configureplatform.cmake

```diff
@@ -1,4 +1,5 @@
 include(CheckPIESupported)
+include(${CMAKE_CURRENT_LIST_DIR}/functions.cmake)
```

Comment:
If we try to invoke `libraries` build on unsupported platform, we get error from line 27 in this file: `clr_unknown_arch` macro undefined, instead of `Only AMD64, ARM64, ARM and I386 are supported` message from that function (defined in `functions.cmake`). Therefore, this inclusion here makes sense (as opposed to including `functions.cmake` before `configureplatform.cmake` at all callsites).